contracts/src/FLK.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import "solmate/tokens/ERC20.sol";
+
+contract FleekToken is ERC20 {
```

Comment:
Should we run this under a proxy contract?

Reply:
We should, but implementation contract shouldnt really be aware its under a proxy in its code. Ive been thinking Proxy/umbrella/governance contract that can manage these as future work

Reply:
Sounds good, as long as we are thinking about upgradability